tasks/config/eslint.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * eslint
+ *
+ * ---------------------------------------------------------------
+ *
+ * # Lints js, jsx, and es6
+ *
+ * For usage docs see:
+ *      https://github.com/sindresorhus/grunt-eslint
+ */
+module.exports = function(grunt) {
+  grunt.config.set('eslint', {
+    target: ['src/*.jsx'],
```

Comment:
Need to update this to point at the correct files. Currently it's not catching anything